### DISCUSSION TITLE
Sensori movimento non rispondono dopo agg al 2022.9.7

### DISCUSSION THREAD

**Discussion by nromagna** [indomus/forum#1957] [HUB: Home Assistant]
Ciao per caso è cambiato qualcosa dopo l'ultimo aggiornamento di HA? Esattamente il 2022.9.7.
Ossia ho qualche sensore Sonoff di movimento che sino alla versione precendete funzionavano alla grande, dopo quest'utimo aggiornamento non vanno più.
Sull'app ewelink ovviamente li vedo attivi e funzionanti.
Grazie.

**Comment by snakuzzo** [comment]
https://github.com/home-assistant/core/releases/tag/2022.9.7

**Reply by nromagna** [reply]
Era un si, un no, oppure un non lo so neppure io??? Grazie

**Reply by snakuzzo** [reply]
era un "vai a quel link e leggi cos'è cambiato con la .7".
Se hai problemi ci sono i log.
Se non ci sono i log vai sulle issue di core e segnala qual è il tuo problema aprendone una nuova

**Reply by snakuzzo** [reply]
P.S. e cortesemente modifica il titolo di questa discussion perché con quello che hai usato nessuno può capire di cosa si tratta
